sdk/spring/scripts/sync_external_dependencies.py

```diff
@@ -5,8 +5,9 @@
 # How to use this script.
 #  1. Make sure file(`.\sdk\spring\spring_boot_SPRING_BOOT_VERSION_managed_external_dependencies.txt`) exist. If it doesn't exist, please run
 #    `.\sdk\spring\scripts\get_spring_boot_managed_external_dependencies.py` to create that file.
-#  2. Update `SPRING_BOOT_VERSION` in this script manually.
-#  3. Run command `python .\sdk\spring\scripts\sync_external_dependencies.py`.
+#  2. Get `SPRING_BOOT_VERSION`.
```

Comment:
1. Add description about how to get the version. Like sdk/spring/scripts/README.md
2. This should before `Make sure file(`.\sdk\spring\spring_boot_SPRING_BOOT_VERSION_managed_external_dependencies.txt`) exist`.